test/infrastructure/docker/internal/docker/machine.go

```diff
@@ -212,40 +208,54 @@ func (m *Machine) Create(ctx context.Context, image string, role string, version
 	if m.container == nil {
 		var err error
 
-		machineImage := m.machineImage(version)
+		// Get the kind kindMapping for the target K8s version.
```

Comment:
```suggestion
		// Get the kindMapping for the target K8s version.
```
nit: maybe (?) Same in the next line